src/components/Header.vue

```diff
@@ -1,8 +1,34 @@
+<script lang="ts">
+import Vue from 'vue';
+import PreferencesButton from '@/components/PreferencesButton.vue';
+
+export default Vue.extend({
+  name:       'rd-header',
```

Comment:
Do this to prevent the new prefs button from showing up in the absence of the feature flag:
```

export default Vue.extend({
  name: 'rd-header',
  ...(
    process.env.RD_MODAL_PREFERENCES === '1' ? {
      components: { PreferencesButton },
      computed:   {
        showPreferences() {
          return this.$config.showPreferences;
        }
      },
      methods:    {
        openPreferences() {
          this.$emit('open-preferences');
        }
      }
    } : {}
  )
});
```